content/doc/book/system-administration/monitoring.adoc

```diff
@@ -22,6 +22,11 @@ See the jira:WEBSITE-738[] EPIC for tasks related to this page, contributions ar
 - plugin:metrics-datadog[Metrics-Datadog plugin for Jenkins]
 - https://www.datadoghq.com/blog/monitor-jenkins-datadog[Jenkins on Datadog]
 
+== Monitoring with Newrelic
+
+- https://opensource.newrelic.com/projects/newrelic/nr-jenkins-plugin[Jenkins on Newrelic]
```

Comment:
why isn't this hosted on the official update center? asking people to build and manually install a plugin is a poor user experience:
https://github.com/newrelic/nr-jenkins-plugin#installation

Reply:
I agree with @timja that it would be much nicer for Jenkins users and New Relic users if they could see the New Relic plugin in their Jenkins update center.  That will allow them to download and install it with a single click and it will assure that any dependent plugins (like credentials and token-macro) are installed and available.

See the instructions to request hosting at https://www.jenkins.io/doc/developer/publishing/requesting-hosting/

I don't see any proprietary dependencies in the plugin pom file and the top level Apache license meets the requirements of the Jenkins project as far as I know.